config.json

```diff
@@ -1033,7 +1033,10 @@
       "core": false,
       "difficulty": 9,
       "slug": "pythagorean-triplet",
-      "topics": null,
+      "topics": [
+		"mathematics",
+		"logic"
```

Comment:
Great suggestions! I would also add `integers` and `lists` since you have to use both to solve the exercise :)

A couple of other minor things, the topics should be indented two spaces in from `topics` and the closing square bracket should be indented in line with `topics` and `unlocked_by`. Does that make sense?
And the topics should be sorted alphabetically :)

Reply:
I thought about adding data structures but I didn't want to limit them. Eg add lists but they want to use trees. Although I do agree most people would use lists. What do you think?

Ah I'll sort alphabetically,

Reply:
The [tests](https://github.com/exercism/java/blob/master/exercises/pythagorean-triplet/src/test/java/PythagoreanTripletTest.java) expect a list to be returned so you do have to use lists in some way to solve the exercise :) And the tests also expect you to use integers.

The way we look at the topics is to see them as things the user either definitely has to use to solve the exercise, or might want to use to solve the exercise. So it doesn't have to only be things they have to use. The only topics we don't want are things that are completely unrelated or things that we think would only be used in a relatively 'bad' solution.

Let me know if you disagree with any of that or if it didn't make sense :)

Great, thanks :)

Reply:
Ah that makes sense, will update.